src/ol/format/IIIFInfo.js

```diff
@@ -40,7 +40,7 @@ import {includes} from '../array.js';
  */
 
 /**
- * @typedef {Object<string,string|number|Array<number|string|IiifProfile>|Object<string, number>|TileInfo>}
+ * @typedef {Object<string,string|number|Array<number|string|IiifProfile>|Object<string, number>|Array<TileInfo>>}
```

Comment:
```suggestion
 * @typedef {Object<string,string|number|Array<number|string|IiifProfile|Object<string, number>|TileInfo>>}
```

Comparing with the state before the faulty commit this might be the desired typing. `sizes` seemed to be an array as well, before.

Reply:
This seems to be correct after the cited spec as well: https://iiif.io/api/image/3.0/#53-sizes

Reply:
The type would be a little stricter in your version, though. You could also keep your version and just add another `Array` around `Object<string,number>` because they can not get mixed.

Reply:
I am good with your proposal. Thank you.